compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -375,8 +375,7 @@
     private static final ParserRuleContext[] TABLE_KEY_RHS =
             { ParserRuleContext.COMMA, ParserRuleContext.CLOSE_PARENTHESIS };
 
-    private static final ParserRuleContext[] ERROR_TYPE_PARAMS =
-            { ParserRuleContext.INFERRED_TYPE_DESC, ParserRuleContext.TYPE_DESC_IN_ANGLE_BRACKETS };
+    private static final ParserRuleContext[] ERROR_TYPE_PARAMS = { ParserRuleContext.TYPE_DESC_IN_ANGLE_BRACKETS };
```

Comment:
We only define an alternative array here when there is more than one path. 
We can get rid of both `ERROR_TYPE_PARAM_START` and `ERROR_TYPE_PARAMS`.

Instead can give next rule for error keyword as `ParserRuleContext.LT`. 
https://github.com/ballerina-platform/ballerina-lang/blob/7d00ba7caf0ad0f40515b7ba9283b03a897f5a49/compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java#L4863-L4866